man/systemd.netdev.xml

```diff
@@ -1979,6 +1982,94 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[BatmanAdvanced] Section Options</title>
+    <para>The [BatmanAdvanced] section only applies for
+    netdevs of kind <literal>batadv</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>GatewayMode=</varname></term>
+        <listitem>
+          <para>The GatewayMode is a string of either <literal>off</literal>, <literal>server</literal> or
+          <literal>client</literal>. A batman-adv node can either run in server mode (sharing its internet
+          connection with the mesh) or in client mode (searching for the most suitable internet connection
+          in the mesh) or having the gateway support turned off entirely (which is the default setting).
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Aggregation=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables aggregation of originator messages. Defaults to
+           true.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>BridgeLoopAvoidance=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables avoidance of loops on bridges. Defaults to true.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>DistributedArpTable=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables the distributed arp table. Defaults to true.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Fragmentation=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables fragmentation. Defaults to true.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>HopPenalty=</varname></term>
+        <listitem>
+          <para>The hop penalty setting allows to modify <command>batctl</command> preference for multihop
```

Comment:
maybe make reference to batctl a proper links: 

```
<citerefentry><refentrytitle>batctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
```

(assuming man page for that are in section 1, you get the idea)